.github/workflows/docker-release.yml

```diff
@@ -0,0 +1,48 @@
+name: Docker Release
+
+on:
+  release:
+    types:
+      - created
+
+jobs:
+  release-docker:
+    name: Build and Publish
+    runs-on: ubuntu-latest
+    steps:
+      - 
+        name: Checkout
+        uses: actions/checkout@v2
+      - 
+        name: Extract Tag Name
+        shell: bash
+        run: echo "##[set-output name=tag;]$(echo ${GITHUB_REF#refs/tags/})"
+        id: extract_tag
+      - 
+        name: Extract Repo Owner
+        shell: bash
+        run: echo "##[set-output name=owner;]$(echo ${GITHUB_REPOSITORY%%/*} | tr '[:upper:]' '[:lower:]')"
+        id: extract_owner
```

Comment:
Although `${{ github.repository_owner }}` exists, it can provide upper case chars that result in the tag being invalid to the GitHub container registry (at present). Although in the case of the mininet repo, this isn't strictly required, it can help out people who fork and protect against any future name changes.

Same goes for `.github/workflows/docker.yml` lines 17-20.